_join/open-positions/onrr-front-end-engineer.md

```diff
@@ -0,0 +1,94 @@
+
+Every American benefits from the revenues ONRR collects; either directly through payments to Tribes and individual Indian mineral owners, or, indirectly through contributions to the Historic Preservation Fund, the Land and Water Conservation Fund, and the Reclamation Fund, and disbursements to States and the U.S. Treasury.
+
+## Fellowship Information
+
+**ONRR - Innovation Design Team Overview:** 
+The Office of Natural Resources Revenue (ONRR), within the U.S. Department of the Interior, is responsible for collecting, tracking, and verifying federal revenue from natural resources. They also manage hundreds of important data sets about natural resource extraction.
+
+ONRR is building an Innovation Design Team to help make this crucial data more open and useful, with the lofty goal of educating and informing public debate around natural resource management on public land.
+
+
+## Role summary:
+
+### Front End Engineer
+
+The Innovation Design Team will manage the U.S. Extractive Industries Transparency Initiative (USEITI) data portal and expand work begun in collaboration with 18F, as well as extending those efforts to other datasets and initiatives throughout ONRR. The Front End Engineer should be well-versed in modern development practices and data tools, utilizing: Sass, Jekyll, JavaScript and D3, SQLite, data APIs, and Node.js.
+
+**Key Objective #1: You will contribute high-quality, well-tested, maintainable code across an entire project lifecycle, using modern software development best practices.** 
+- Practice and enthusiastically share agile methodologies throughout all stages of the project lifecycle.
+- Leverage front end technologies and data transformation methods to parse and juggle records from multiple, sometimes disorganized data sets. 
+- Participate in code review, architecture discussions, and feature prioritization.
+- Take pride of ownership in all projects you touch; leave code better than you found it. 
+- Contribute to documentation, tests, style fixes, accessibility, performance, security, etc.
+- Deliver projects that are easy to deploy, update, and monitor by ensuring the tooling for this is present early in the project development cycle.
+- Use usability research, analytics, and other metrics to influence project planning, design, and development.
+
+
+**Objective #2: You will find and create tools and resources to help ONRR address their data management needs.** 
+- Acquire broad knowledge of the landscape of past and present data management challenges that exist within ONRR. 
+- Recommend existing data-management tools and resources based on ONRR’s unique needs
```

Comment:
Need a "." at the end of the line to maintain consistency. Same with the next point as well.